README
NAME
    Task::BeLike::Shantanu - All my default Modules in a perl installation

VERSION
    version 0.09

SUPPORT
  Bugs / Feature Requests
    Please report any bugs or feature requests through github at
    <https://github.com/shantanubhadoria/task-belike-shantanu/issues>. You
    will be notified automatically of any progress on your issue.

  Source Code
    This is open source software. The code repository is available for
    public review and contribution under the terms of the license.

    <https://github.com/shantanubhadoria/task-belike-shantanu>

      git clone git://github.com/shantanubhadoria/task-belike-shantanu.git

AUTHOR
    Shantanu Bhadoria <shantanu@cpan.org>

CONTRIBUTOR
    Shantanu <shantanu@cpan.org>

COPYRIGHT AND LICENSE
    This software is copyright (c) 2013 by Shantanu Bhadoria.

    This is free software; you can redistribute it and/or modify it under
    the same terms as the Perl 5 programming language system itself.



Changes
Changelog for Task-BeLike-Shantanu

0.08 2013-10-16T04:30:43
 - v0.08 [Shantanu Bhadoria]
 - Added Pod::Simple to pass kwalitee tests properly [Shantanu Bhadoria]

0.07 2013-09-15T04:40:03
 - v0.07
   
    - remove Printer::Thermal [Shantanu]
    - refactor distro [Shantanu]
    - Merge branch 'master' of
      github.com:shantanubhadoria/Task-BeLike-Shantanu [Shantanu]
    - refactor distro [Shantanu]
    - Bump Changes [Shantanu Bhadoria] [Shantanu]
 - remove Printer::Thermal [Shantanu]
 - refactor distro [Shantanu]
 - Merge branch 'master' of
   github.com:shantanubhadoria/Task-BeLike-Shantanu [Shantanu]
 - refactor distro [Shantanu]
 - Bump Changes [Shantanu Bhadoria]

0.06 2013-05-23T06:15:51
 - v0.06
   
	   - 2013-05-23 14:14:49 Asia/Singapore [Shantanu Bhadoria]
 - Added Net::SSLeay to prereqs [Shantanu Bhadoria]
 - Bump Changes [Shantanu Bhadoria]

0.05 2013-03-05T15:21:07
 - v0.05
   
	   - 2013-03-05 23:20:32 Asia/Singapore [Shantanu Bhadoria]
 - Removed unused modules [Shantanu Bhadoria]

0.04 2013-03-05T15:02:00
 - Adding Changes [Shantanu Bhadoria]
 - Moved to Shantanu Plugin bundle for Dist::Zilla [Shantanu Bhadoria]

0.004 2013-03-05T14:54:40
 - Moved to Shantanu Plugin bundle for Dist::Zilla [Shantanu Bhadoria]
 - adding Gitignore [Shantanu Bhadoria]
 - Added Version Tag [Shantanu Bhadoria]
 - Adding Basic Files to build dependencies for the Distro [Shantanu
   Bhadoria]
 - First Commit [Shantanu Bhadoria]



lib/Task/BeLike/Shantanu.pm
use strict;
use warnings;

package Task::BeLike::Shantanu;

# PODNAME: Task::BeLike::Shantanu
# ABSTRACT: All my default Modules in a perl installation
our $VERSION = '0.09'; # VERSION
# Dependencies

use 5.010;
use Acme::CPANAuthors::India;
use Dist::Zilla::PluginBundle::SHANTANU;
use File::UStore;
use Pod::Weaver::PluginBundle::SHANTANU;

use autodie;
use App::cpanminus;
use App::cpanoutdated;
use Authen::Passphrase;
use Catalyst 5.90000;
use Catalyst::Plugin::Assets;
use Catalyst::Plugin::Authentication;
use Catalyst::Plugin::Session;
use Catalyst::Plugin::Session::Store::FastMmap;
use Catalyst::Plugin::StatusMessage;
use Catalyst::Runtime;
use Data::Dumper 2.14;
use DateTime::Format::MySQL;
use Device::SerialPort;
use DBIx::Class;
use DBIx::Class::PassphraseColumn;
use DBIx::Class::Validation;
use DBIx::Class::InflateColumn::Authen::Passphrase;
use Digest::MD5;
use Dist::Zilla 4.300000;
use ExtUtils::MakeMaker 6.60;
use ExtUtils::ParseXS 3.10;
use File::ChangeNotify 0.23;
use File::Copy 2.20;
use File::Find::Rule 0.33;
use File::HomeDir 1.00;
use File::Spec 3.40;
use HTML::FormFu::Constraint::DBIC::Unique;
use IO::Handle;
use IO::Socket;
use JSON::XS;
2.33;
use Log::Log4perl 1.40;
use Moose 2.06;
use Net::IP;
use Net::SSLeay;
use Perl::Critic;
use Perl::Tidy;
use Test::Pod 1.48;
use Pod::Simple 3.28;
use Starman 0.30;
use YAML 0.84;
use YAML::XS 0.38;

1;

__END__

=pod

=head1 NAME

Task::BeLike::Shantanu - All my default Modules in a perl installation

=head1 VERSION

version 0.09

=for :stopwords cpan testmatrix url annocpan anno bugtracker rt cpants kwalitee diff irc mailto metadata placeholders metacpan

=head1 SUPPORT

=head2 Bugs / Feature Requests

Please report any bugs or feature requests through github at 
L<https://github.com/shantanubhadoria/task-belike-shantanu/issues>.
You will be notified automatically of any progress on your issue.

=head2 Source Code

This is open source software.  The code repository is available for
public review and contribution under the terms of the license.

L<https://github.com/shantanubhadoria/task-belike-shantanu>

  git clone git://github.com/shantanubhadoria/task-belike-shantanu.git

=head1 AUTHOR

Shantanu Bhadoria <shantanu@cpan.org>

=head1 CONTRIBUTOR

Shantanu <shantanu@cpan.org>

=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2013 by Shantanu Bhadoria.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.

=cut


Makefile.PL
# This Makefile.PL for  was generated by Dist::Zilla.
# Don't edit it but the dist.ini used to construct it.

use strict;
use warnings;

use 5.010;

use ExtUtils::MakeMaker 6.30;



my %WriteMakefileArgs = (
  "ABSTRACT" => "All my default Modules in a perl installation",
  "AUTHOR" => "Shantanu Bhadoria <shantanu\@cpan.org>",
  "BUILD_REQUIRES" => {},
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => "6.30"
  },
  "DISTNAME" => "Task-BeLike-Shantanu",
  "EXE_FILES" => [],
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.010",
  "NAME" => "Task::BeLike::Shantanu",
  "PREREQ_PM" => {
    "Acme::CPANAuthors::India" => 0,
    "App::cpanminus" => 0,
    "App::cpanoutdated" => 0,
    "Authen::Passphrase" => 0,
    "Catalyst" => "5.90000",
    "Catalyst::Plugin::Assets" => 0,
    "Catalyst::Plugin::Authentication" => 0,
    "Catalyst::Plugin::Session" => 0,
    "Catalyst::Plugin::Session::Store::FastMmap" => 0,
    "Catalyst::Plugin::StatusMessage" => 0,
    "Catalyst::Runtime" => 0,
    "DBIx::Class" => 0,
    "DBIx::Class::InflateColumn::Authen::Passphrase" => 0,
    "DBIx::Class::PassphraseColumn" => 0,
    "DBIx::Class::Validation" => 0,
    "Data::Dumper" => "2.14",
    "DateTime::Format::MySQL" => 0,
    "Device::SerialPort" => 0,
    "Digest::MD5" => 0,
    "Dist::Zilla" => "4.300000",
    "Dist::Zilla::PluginBundle::SHANTANU" => 0,
    "ExtUtils::MakeMaker" => "6.60",
    "ExtUtils::ParseXS" => "3.10",
    "File::ChangeNotify" => "0.23",
    "File::Copy" => "2.20",
    "File::Find::Rule" => "0.33",
    "File::HomeDir" => "1.00",
    "File::Spec" => "3.40",
    "File::UStore" => 0,
    "HTML::FormFu::Constraint::DBIC::Unique" => 0,
    "IO::Handle" => 0,
    "IO::Socket" => 0,
    "JSON::XS" => 0,
    "Log::Log4perl" => "1.40",
    "Moose" => "2.06",
    "Net::IP" => 0,
    "Net::SSLeay" => 0,
    "Perl::Critic" => 0,
    "Perl::Tidy" => 0,
    "Pod::Simple" => "3.28",
    "Pod::Weaver::PluginBundle::SHANTANU" => 0,
    "Starman" => "0.30",
    "Test::Pod" => "1.48",
    "YAML" => "0.84",
    "YAML::XS" => "0.38",
    "autodie" => 0,
    "strict" => 0,
    "warnings" => 0
  },
  "TEST_REQUIRES" => {
    "File::Find" => 0,
    "File::Spec::Functions" => 0,
    "File::Temp" => 0,
    "List::Util" => 0,
    "Test::More" => 0
  },
  "VERSION" => "0.09",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  my $tr = delete $WriteMakefileArgs{TEST_REQUIRES};
  my $br = $WriteMakefileArgs{BUILD_REQUIRES};
  for my $mod ( keys %$tr ) {
    if ( exists $br->{$mod} ) {
      $br->{$mod} = $tr->{$mod} if $tr->{$mod} > $br->{$mod};
    }
    else {
      $br->{$mod} = $tr->{$mod};
    }
  }
}

unless ( eval { ExtUtils::MakeMaker->VERSION(6.56) } ) {
  my $br = delete $WriteMakefileArgs{BUILD_REQUIRES};
  my $pp = $WriteMakefileArgs{PREREQ_PM};
  for my $mod ( keys %$br ) {
    if ( exists $pp->{$mod} ) {
      $pp->{$mod} = $br->{$mod} if $br->{$mod} > $pp->{$mod};
    }
    else {
      $pp->{$mod} = $br->{$mod};
    }
  }
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);



